igvm/cli.py

```diff
@@ -185,6 +185,14 @@ def parse_args():
         action='store_true',
         help='Build or migrate VM only to a HV with the same environment of VM'
     )
+    subparser.add_argument(
+        '--disk-size',
+        dest='disk_size',
+        default=0,
+        type=int,
+        help='When --offline-transport=xfs is used, this parameter may specify'
```

Comment:
It is an optional argument, once it is specified it does things always. Saying "may" means that it could still be ignored depending on other conditions.